analytics/tests/test_views.py

```diff
@@ -825,7 +827,11 @@ def check_realm_reactivation_link_query_result(result: HttpResponse) -> None:
             result = self.client_get("/activity/support", {"q": email})
             check_realm_creation_query_result(result, email)
 
-            do_create_multiuse_invite_link(self.example_user("hamlet"), invited_as=1)
+            do_create_multiuse_invite_link(
+                self.example_user("hamlet"),
+                invited_as=1,
+                invite_expires_in_days=invite_expires_in_days,
```

Comment:
Also, do you need to pass this parameter for all of these tests?  I'd imagine that the default would be correct in some of them.